templates/launch.html

```diff
@@ -21,12 +22,44 @@ <h1>Launch a survey</h1>
         </select>
     </div>
 
+    <p>----------</p>
+
+    <strong><u>Launch by Schema URL</u></strong>
+    <div class="field-container">
+        <span>
+            <label for="schema-url-survey-type">Survey Type</label>
+            <select name="survey-types" id="schema-url-survey-type">
+                <option selected disabled>Select Survey Type</option>
+                {{ range $surveyType, $schemasList := .Schemas }}
+                <option name="{{ $surveyType }}" value="{{ $surveyType }}" data-schema-url-survey-type="{{ $surveyType }}">{{ $surveyType }}</option>
+            {{ end }}
+            </select>
+        </span>
+    </div>
+
+    <div class="field-container">
+        <span>
+            <label for="schema-url">Schema URL</label>
+            <input id="schema-url" name="survey_url" type="text" class="qa-schema_url">
+            <input type="submit" onClick="return loadMetadataForSchemaUrl()" value="Load Schema" class="qa-btn-submit-dev btn" id="schema-url-btn"/>
```

Comment:
Can we add similar window alert (onclick event) to other submit buttons on this page since we are making changes here? Currently when no schema is selected and we press submit button we are not getting prompted.

Reply:
I've updated the CSS, the buttons are disabled to start with, so the colour has been updated. You will need to clear cache to see the change.